[2022-04-05 22:49:01,033] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_posts_answers.ingest_posts_answers_to_bucket scheduled__2009-07-01T00:00:00+00:00 [queued]>
[2022-04-05 22:49:01,332] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_posts_answers.ingest_posts_answers_to_bucket scheduled__2009-07-01T00:00:00+00:00 [queued]>
[2022-04-05 22:49:01,333] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-05 22:49:01,333] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-05 22:49:01,333] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-05 22:49:01,500] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): processing_tasks_posts_answers.ingest_posts_answers_to_bucket> on 2009-07-01 00:00:00+00:00
[2022-04-05 22:49:01,522] {standard_task_runner.py:52} INFO - Started process 17308 to run task
[2022-04-05 22:49:01,678] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'IngestToGCP', 'processing_tasks_posts_answers.ingest_posts_answers_to_bucket', 'scheduled__2009-07-01T00:00:00+00:00', '--job-id', '8652', '--raw', '--subdir', 'DAGS_FOLDER/ingest_dag.py', '--cfg-path', '/tmp/tmp8o1iqo3a', '--error-file', '/tmp/tmpkajat6ux']
[2022-04-05 22:49:01,680] {standard_task_runner.py:80} INFO - Job 8652: Subtask processing_tasks_posts_answers.ingest_posts_answers_to_bucket
[2022-04-05 22:49:02,823] {logging_mixin.py:109} INFO - Running <TaskInstance: IngestToGCP.processing_tasks_posts_answers.ingest_posts_answers_to_bucket scheduled__2009-07-01T00:00:00+00:00 [running]> on host 9fe4286b2855
[2022-04-05 22:49:03,377] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-05 22:49:03,614] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=IngestToGCP
AIRFLOW_CTX_TASK_ID=processing_tasks_posts_answers.ingest_posts_answers_to_bucket
AIRFLOW_CTX_EXECUTION_DATE=2009-07-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2009-07-01T00:00:00+00:00
[2022-04-05 22:49:05,926] {python.py:175} INFO - Done. Returned value was: None
[2022-04-05 22:49:06,906] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=IngestToGCP, task_id=processing_tasks_posts_answers.ingest_posts_answers_to_bucket, execution_date=20090701T000000, start_date=20220405T224901, end_date=20220405T224906
[2022-04-05 22:49:07,280] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-05 22:49:07,866] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
